hexadecimal/solution.py

```diff
@@ -1,10 +1,17 @@
-from functools import reduce
+_hex_code = dict(zip('0123456789', range(10)))
+_hex_code.update(zip('abcdef', range(10, 16)))
+_hex_code.update(zip('ABCDEFG', range(10, 16)))
 
 
 def hexa(s):
```

Comment:
Personally I prefer a bit more descriptive Variable names :-)

Reply:
would ``hexdigit_to_int`` sound better?

Reply:
That sounds good, although I meant the single letter names `s` and `c` :-)